css-ecommerce-page/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="col-full">
+        <h2>New Arrivals</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-fifth">
+        <img class="image" src="images/dress1.jpeg">
+        <div class="inline">
+          <p class="bold">Free People</p>
+          <p class="t-pad grey">Size 6</p>
+        </div>
```

Comment:
Rather than making this section with a `div` holding `p` elements as children, you should make this with `p` elements that hold `span` elements as children.